target/linux/ramips/dts/mt7621_iodata_wn-dx1167r.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_iodata_wn-xx-xr.dtsi"
```

Comment:
I do not really see where the "xx-xr" comes from, but I also do not care to much about it...

Reply:
- WN-AX1167GR2
- WN-AX2033GR
- WN-AX2033GR2
- WN-DX1167R

The similar scheme is used for WN-AC1167DGR/WN-AC1600DGRx in ath79 target.

Reply:
I'm not referring to the fact the you use placeholder characters there, but to how they are chosen. I just do not see how "WN-AX1167GR2" translates in "wn-xx-xr".
However, the different names do not really make it easy to find an obvious solution, so you as well keep yours.